cg/constants/demultiplexing.py

```diff
@@ -61,9 +61,9 @@ class SampleSheetHeaderColumnNames(StrEnum):
 OPTION_BCL_CONVERTER = click.option(
     "-b",
     "--bcl-converter",
-    type=click.Choice(["bcl2fastq", "dragen"]),
+    type=click.Choice(["bcl2fastq", "dragen", "bcl_convert"]),
```

Comment:
Remove "bcl_convert" for now, we will remove the bcl conversion software flag in a separate PR anyway.